packages/ethereum-multicall/src/utils.ts

```diff
@@ -4,6 +4,8 @@ export class Utils {
    * @param object The object
    */
   public static deepClone<T>(object: T): T {
-    return JSON.parse(JSON.stringify(object)) as T;
+    const cloned = JSON.parse(JSON.stringify(object)) as T;
```

Comment:
hopefuilly they fixed this issue since we could end up with entangled objects that shouldn't be